Modules/Microsoft365DSC/DSCResources/MSFT_AADRoleSetting/MSFT_AADRoleSetting.psm1

```diff
@@ -0,0 +1,1232 @@
+                EligibleAlertNotificationOnlyCritical                       = $EligibleAlertNotificationOnlyCritical
+                EligibleAssigneeNotificationDefaultRecipient                = $EligibleAssigneeNotificationDefaultRecipient
+                EligibleAssigneeNotificationAdditionalRecipient             = [System.String[]]$EligibleAssigneeNotificationAdditionalRecipient
+                EligibleAssigneeNotificationOnlyCritical                    = $EligibleAssigneeNotificationOnlyCritical
+                EligibleApproveNotificationDefaultRecipient                 = $EligibleApproveNotificationDefaultRecipient
+                EligibleApproveNotificationAdditionalRecipient              = [System.String[]]$EligibleApproveNotificationAdditionalRecipient
+                EligibleApproveNotificationOnlyCritical                     = $EligibleApproveNotificationOnlyCritical
+                ActiveAlertNotificationDefaultRecipient                     = $ActiveAlertNotificationDefaultRecipient
+                ActiveAlertNotificationAdditionalRecipient                  = [System.String[]]$ActiveAlertNotificationAdditionalRecipient
+                ActiveAlertNotificationOnlyCritical                         = $ActiveAlertNotificationOnlyCritical
+                ActiveAssigneeNotificationDefaultRecipient                  = $ActiveAssigneeNotificationDefaultRecipient
+                ActiveAssigneeNotificationAdditionalRecipient               = [System.String[]]$ActiveAssigneeNotificationAdditionalRecipient
+                ActiveAssigneeNotificationOnlyCritical                      = $ActiveAssigneeNotificationOnlyCritical
+                ActiveApproveNotificationDefaultRecipient                   = $ActiveApproveNotificationDefaultRecipient
+                ActiveApproveNotificationAdditionalRecipient                = [System.String[]]$ActiveApproveNotificationAdditionalRecipient
+                ActiveApproveNotificationOnlyCritical                       = $ActiveApproveNotificationOnlyCritical
+                EligibleAssignmentAlertNotificationDefaultRecipient         = $EligibleAssignmentAlertNotificationDefaultRecipient
+                EligibleAssignmentAlertNotificationAdditionalRecipient      = [System.String[]]$EligibleAssignmentAlertNotificationAdditionalRecipient
+                EligibleAssignmentAlertNotificationOnlyCritical             = $EligibleAssignmentAlertNotificationOnlyCritical
+                EligibleAssignmentAssigneeNotificationDefaultRecipient      = $EligibleAssignmentAssigneeNotificationDefaultRecipient
+                EligibleAssignmentAssigneeNotificationAdditionalRecipient   = [System.String[]]$EligibleAssignmentAssigneeNotificationAdditionalRecipient
+                EligibleAssignmentAssigneeNotificationOnlyCritical          = $EligibleAssignmentAssigneeNotificationOnlyCritical
+                ApplicationId                                               = $ApplicationId
```

Comment:
These last five parameters (ApplicationId to Credential) are not necessary to be returned from the Get method and should be removed. They only add additional lines and are never used.